packages/components/src/components/toolbar.tsx

```diff
@@ -1,11 +1,36 @@
-import { useAtom } from 'jotai'
+import { useAtom, atom } from 'jotai'
+import { Docs } from './docs'
 import { configAtom } from './provider'
+import { Settings } from "./settings"
+
+const showDocsAtom = atom(false),
+  showSettingsAtom = atom(false)
```

Comment:
these should be two separate declarations

Reply:
done